overlord/devicestate/handlers_install.go

```diff
@@ -315,10 +316,22 @@ func checkEncryption(model *asserts.Model) (res bool, err error) {
 	// encryption is required in secured devices and optional in other grades
 	if err := secbootCheckKeySealingSupported(); err != nil {
```

Comment:
Slightly related, can secboot.CheckKeySealingSupported properly distinguish between the cases when there's no TPM device or there is one but talking to the TPM fails. Where, in the latter case we should probably throw some error during install?